src/views/catalog/utils/quantity.ts

```diff
@@ -1,5 +1,8 @@
 import byteSize, { ByteSizeResult } from 'byte-size';
 
+const fixUnitSuffix = (unit: string): string =>
+  unit.charAt(unit.length - 1) === 'B' ? unit : unit + 'B';
+
 const multipliers: Record<string, number> = {};
 multipliers.B = 1;
 multipliers.Ki = multipliers.B * 1024;
```

Comment:
why declare such a way? 
multipliers = {
B: 1,
Ki: 1024
};

Reply:
Good question. Ask Ugo... he wrote it. =)

Reply:
@upalatucci

Reply:
Yes we can follow the matan suggestion.
Was more complicated before 😂 I thought could be much more readable this way

Reply:
We can use the `units: 'iec'` instead of this custom unit at this point